apps/evaluations/forms.py

```
@@ -593,3 +593,38 @@ def clean_name(self):
+    new_dataset_name = forms.CharField(label="Create new dataset", required=False)
+    message_ids = forms.CharField(required=True, widget=forms.HiddenInput(attrs={"x-model": "selectedMessages"}))
+
+    def __init__(self, team, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        dataset_choices = [("", "---")] + [(dataset.id, dataset.name) for dataset in team.evaluationdataset_set.all()]
+        self.fields["dataset"].choices = dataset_choices
+
+    def clean_message_ids(self):
+        message_ids = self.cleaned_data["message_ids"]
+        if not message_ids or not message_ids.strip():
+            raise forms.ValidationError("Please select at least one message.")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)